README.md

```diff
@@ -52,8 +52,12 @@ Chat Room
 
 Want to chat with other members of the PowerShell community?
 
+We have a Gitter Room which you can join below
```

Comment:
I'm still getting used to how Markdown ends up getting displayed. Need to make use of VSCode's Markdown Preview extension a bit more.